lib/registrant.rb

```diff
@@ -0,0 +1,22 @@
+class Registrant
+
+  attr_accessor :name, :age, 
+                :permit, :license_data
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @license_data = {written: false, 
+    license: false, renewed: false}
+    @permit = permit  
+  end
+
+  def permit?
+    @permit == true
+  end
+  
+  def earn_permit
+    @permit = true
+  end
+
```

Comment:
👍 This class looks good. I would, however, remove `:permit` from the accessors since we don't need to ever call `.permit` on a Registrant object. On that subject also, I think only `license_data` is the attribute that should be an accessor since it can be updated from other classes. All others (name and age) can just be set as readers.